csw-location/src/main/scala/csw/services/location/internal/Registry.scala

```diff
@@ -9,48 +9,45 @@ import csw.services.location.models.{Connection, Location}
 import scala.concurrent.duration.DurationDouble
 
 /**
-  * A `Registry` that manages [[akka.cluster.ddata.Replicator.Update]] and [[akka.cluster.ddata.Replicator.Get]] messages
-  * for both cluster data types namely, [[akka.cluster.ddata.LWWRegister]] and [[akka.cluster.ddata.LWWMap]]
+  * Registry is used to create CRDTs and manage its update and get messages for replicator
   *
-  * @param Key        The `Key` against which `update` or `get` will be performed
-  * @param EmptyValue The initial value passed to [[akka.cluster.ddata.Replicator.Update]]
-  * @tparam K The type of `Key`
-  * @tparam V The type of `ReplicatedData`
+  * @param Key        The CRDT key
+  * @param EmptyValue The default value of CRDT key
+  * @tparam K The type of CRDT key
+  * @tparam V The type of ReplicatedData
   */
 class Registry[K <: Key[V], V <: ReplicatedData](val Key: K, val EmptyValue: V) {
   /**
-    * Creates [[akka.cluster.ddata.Replicator.Update]] with the `Key`, `EmptyValue` and function `f` which will update
-    * majority nodes in cluster
+    * Creates an update message for replicator with WriteConsistency as majority
     *
+    * @see [[akka.cluster.ddata.Replicator.Update]]
     * @param f A callback function which is passed to Replicator.Update
     */
   def update(f: V ⇒ V): Update[V] = Update(Key, EmptyValue, WriteMajority(5.seconds))(f)
 
   /**
-    * Creates [[akka.cluster.ddata.Replicator.Get]] with `Key` which will read from local copy of cluster data
+    * Creates a get message for replicator with ReadConsistency as Local
```

Comment:
My words: Unlike for update, read value it returned from the local cache of the node.